pkg/util/config/config.go

```diff
@@ -43,6 +43,7 @@ type Resources struct {
 	MaxInstances            int32             `name:"max_instances" default:"-1"`
 	MinInstances            int32             `name:"min_instances" default:"-1"`
 	ShmVolume               *bool             `name:"enable_shm_volume" default:"true"`
+	NodeSelector            map[string]string `name:"node_selector" default:""`
```

Comment:
You introduced a new field to the configuration, but it's not reflected in other [files](https://github.com/zalando/postgres-operator/blob/master/docs/developer.md#introduce-additional-configuration-parameters). On the other hand, we already have the `NodeReadinesLabel` do define where Pods should be scheduled.